src/Service/Google.php

```diff
@@ -44,13 +56,16 @@ public function getExchangeRate(ExchangeRateQuery $exchangeQuery)
         }
 
         $xpath = new \DOMXPath($document);
-        $nodes = $xpath->query('//span[@class="bld"]');
+        $nodes = $xpath->query('//div[@class="vk_ans vk_bk"]');
 
-        if (0 === $nodes->length) {
+        if (1 !== $nodes->length) {
```

Comment:
It's safer this way, we don't know if it could be more than one node with these two classes.